draft-bbs-signatures.md

```diff
@@ -862,10 +944,16 @@ signature\_dst\_generator\_seed
 : A global seed value of "BBS\_BLS12381G1\_XOF:SHAKE-256\_SSWU\_RO\_SIGNATURE\_DST\_GENERATOR\_SEED" which is used by the [CreateGenerators](#creategenerators) operation to compute the generator used to sign the signature domain separation tag (H_d).
 
 hashing\_elements\_to\_scalars
-: hash\_to\_scalar
+: hash\_to\_scalar.
 
 xof\_no\_of\_bytes
-: 64
+: 64.
+
+octet\_scalar\_length
+: 32, based on the RECOMMENDED approach of `ceil(log2(q)/8)`.
+
+octet\_point\_length
```

Comment:
Isn't this actually specific to the sub-group?

Reply:
if so we should just update the definition

Reply:
It’s not that it’s recommended it’s the actual field size. This is curve dependent so we should specify as a system parameter with the curve

Reply:
We may want to leave it as a recommendation for applications that may want to apply some custom optimization?? Although i do agree that most cases will use that method. It is also true that the number itself is only base field depended so IMO the definition is fine as it is.